dali/pipeline/operators/crop/slice.h

```diff
@@ -20,24 +20,28 @@
 #include "dali/core/common.h"
 #include "dali/core/error_handling.h"
 #include "dali/pipeline/operators/common.h"
+#include "dali/pipeline/operators/crop/slice_attr.h"
 #include "dali/pipeline/operators/crop/slice_base.h"
 #include "dali/pipeline/operators/operator.h"
 
 namespace dali {
 
 template <typename Backend>
-class Slice : public SliceBase<Backend> {
+class Slice : public SliceBase<Backend>, protected SliceAttr {
```

Comment:
Maybe it is a good time to change it from the inheritance to composition?

Reply:
done